internal/paascharm/gostructs.go

```diff
@@ -0,0 +1,142 @@
+package paascharm
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"unicode"
+)
+
+const CommonPrefix = "APP_"
+
+// Map from integration key in charmcraft.yaml to Go name
+var IntegrationsToGoName = map[string]string{
```

Comment:
No need to export (capitalise) variables like this -- they're internal to the package. Only export/capitalise the functions needed externally.